natural-sql-7b/README.md

```diff
@@ -0,0 +1,65 @@
+# NaturalSQL-7B Model
+
+Welcome to the `natural-sql-7b` model documentation. This model is part of the NaturalSQL series by ChatDB, designed to generate SQL queries from natural language questions with state-of-the-art performance.
+
+## Overview
+
+The `natural-sql-7b` model showcases exceptional ability in generating correct SQL queries on novel schemas. It is benchmarked using the SQL-Eval framework and excels in understanding complex query intentions.
+
+## Setup and Dependencies
+
+To use the `natural-sql-7b` model, ensure you have the following dependencies installed:
+
+```bash
+pip install transformers==4.35.2 accelerate sqlparse
+```
+
```

Comment:
Doesn't talk about using a truss at all. It should say something like "cd into truss-examples/natural-sql-7b", then run "truss push".